### ISSUE TITLE
[1.0.18] Samsung Galaxy S4/5.0.1 - app crash on export clip

### ISSUE THREAD

**Issue by ukrpas** [Lomotif/lomotif-android-issues#134]
Steps:
Frequency: 2 / 5
1. Launch Lomotif
2. Click on start
3. Select clip form device gallery
4. Add music from device
5. When clip is ready, press on clip, to stop video.
6. Press on "Export" icon
Act: app was crashed 2 times (you can see logs on Crashlytics, if not, i`ll try to send log using Logcat. One time i see an error "An error has occured while trying to export the Lomotif".

**Comment by ukrpas** [comment]
[Log_2016-04-21_21-25-44.txt](https://github.com/Lomotif/lomotif-android-issues/files/230507/Log_2016-04-21_21-25-44.txt)

**Comment by jpaulo1205** [comment]
@ukrpas do you have a File Explorer app in your device? If so, can you check your SD Card and confirm if there is a DCIM folder? Thanks

**Comment by ukrpas** [comment]
@jpaulo1205 yes, path is \extSdCard\DCIM\Camera

**Comment by jpaulo1205** [comment]
@ukrpas ok i see. is there a Lomotif folder inside the DCIM?

**Comment by ukrpas** [comment]
@jpaulo1205 no. Lomotif folder presents only in device internal storage in DCIM, path like sdcard0\DCIM\Lomotif. I do not see Lomotif folder in the external card  - extSdCard\DCIM

**Comment by jpaulo1205** [comment]
@ukrpas ok. thanks, lastly, can you give me the remaining storage memory for the internal and external card?

**Comment by ukrpas** [comment]
@jpaulo1205 
![screenshot_2016-04-22-07-45-15](https://cloud.githubusercontent.com/assets/11840791/14731940/4395cfa4-085e-11e6-8ad0-ae40b1718c54.png)

**Comment by jpaulo1205** [comment]
@ukrpas thanks for the info. will be investigating further for this issue. could you also check if there is a com.lomotif.android.dev directory in the internal storage?

**Comment by ukrpas** [comment]
Yes, i see com.lomotif.android.dev in Internal storage

Pavel
23 Апр 2016 г. 1:06 пользователь "jpaulo1205" notifications@github.com
написал:

**Comment by vysohanych** [comment]
I can't reproduce it on build 1.0.19.1 on Asus Nexus 7 Android 5.1.1.
